internal/split/split_service.go

```diff
@@ -0,0 +1,88 @@
+package split
+
+import (
+	"io/ioutil"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/splitio/go-client/v6/splitio/client"
+	"github.com/splitio/go-client/v6/splitio/conf"
+
+	configAPI "github.com/newrelic/newrelic-cli/internal/config/api"
+)
+
+// Split.io client API keys
+// These keys are client-facing and are only used to fetch splits.
+// There is no security risk in exposing these keys, as the only purpose they
+// serve is to retrieve experiments and can not be used to modify anything
+// within our internal Split.io system.
+var (
+	prod      = "8me2vu6v8lhssdkrpenp1uunl9s3bdc8njqp"
+	staging   = "mcf9oimts3laqli01e2ktrjdudkdbh8dg42a"
+	accountID = configAPI.GetActiveProfileAccountID()
+)
+
+type Service struct {
+	client *client.SplitClient
+}
+
+// Creates a new instance of a Split Factory
+// Using "localhost" as the apiKey allows us to use Split.io
+// in localhost mode as defined in their documentation
+func NewSplitService(apiKey string, region string) (*Service, error) {
+	cfg := conf.Default()
+	if apiKey == "localhost" {
+		cfg.SplitFile = CreateMockSplits()
```

Comment:
Move assignment to test setup from within a test file?

Reply:
Split requires the `apiKey` portion to be `localhost` for unit-testing. I don't see a good way of extracting the `cfg := conf.Default()` portion from the new service initialization, but let me know what you were thinking!

Reply:
Maybe just export a `SplitFile string` field from the `split` package, and set `cfg.SplitFile = SplitFile` if it is set to anything other than the empty string?  Then we could set that value to the result of `CreateMockSplits()` from inside the test.